newsletter/newsletter.php

```diff
@@ -272,6 +272,9 @@ function get_newsletter($multipart_boundary = null) {
 					$subcategories[$category->term_id][$subcategory->term_id] = $subcategory->name;
 
 					$posts[$category->term_id][$subcategory->term_id][] = get_post_details($post);
+
+					// Shortcodes are being incorrectly interpreted and their content should not be displayed anyway in the newletter excerpt
+					$posts[$category->term_id][$subcategory->term_id][0]['post']->post_content = preg_replace('/\[.+\]/','', $posts[$category->term_id][$subcategory->term_id][0]['post']->post_content);
```

Comment:
What about https://developer.wordpress.org/reference/functions/strip_shortcodes/ ?
It's a bit more complex, but should fit anyway.

Reply:
Right, didn't see that. Furthermore shortcode content is removed anyway (by Twig I guess?), and it's not more complex, so it's perfect.
Done